services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/switches/SwitchRulesSpec.groovy

```diff
@@ -38,13 +38,16 @@ class SwitchRulesSpec extends BaseSpecification {
     @Shared
     Switch srcSwitch, dstSwitch
     @Shared
-    List defaultRules
+    List srcSwDefaultRules
+    @Shared
+    List dstSwDefaultRules
     @Shared
     int flowRulesCount = 2
 
     def setup() {
         (srcSwitch, dstSwitch) = topology.getActiveSwitches()[0..1]
-        defaultRules = northboundService.getSwitchRules(srcSwitch.dpId).flowEntries
+        srcSwDefaultRules = northboundService.getSwitchRules(srcSwitch.dpId).flowEntries
+        dstSwDefaultRules = northboundService.getSwitchRules(dstSwitch.dpId).flowEntries
```

Comment:
I'm not sure whether it's just easier to pull certain rules for certain switch directly in the test rather than pull rules for both switches for every test in the spec. I bet in most cases you only need one of them or neither

Reply:
Yes, you are right. But I don't think that is a heavy operation. Let's have such a variant of fixing the test until I come up with a new solution.